ofl/hind/DESCRIPTION.en_us.html

```diff
@@ -2,6 +2,7 @@
 Hind is an Open Source typeface supporting the Devanagari and Latin scripts. 
 Developed explicitly for use in User Interface design, the Hind font family includes five styles. 
 Hind’s letterforms have a humanist-style construction, which is paired with seemingly monolinear strokes. 
+<<<<<<< HEAD
```

Comment:
We have a merge conflict here. Idk which parts we want to keep. It feels quite long.